llm_bench/python/llm_bench_utils/bench_ldm_sr.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
I do not fully understand logic for splitting benchmarks by files here.
it can be better if it will be strong logically enough and readable in file naming, so I suggest 2 variants how to better organize that:
* by modality (image (image generation, ldm super resolution) /tex (llms text generation) )/audio (in future whisper, tts models)/multimodality (in future for visiual-language models like llava))

by task (text-generation, image-generation, super_resolution, asr e.t.c.)

@wgzintel  @peterchen-intel  which from these notation will be better?

Reply:
also not sure that llm_bench_utils is right place for that, it contains a lot of different utilities, while benchmarking code is important part of benchmark and should be visible for users ( it is not good to mix it with other utilities, maybe better to have separated namespace for that (e.g. from use_cases/text_generation.py), possibly out of llm_benhc_utils

Reply:
@eaidova  organized by task and move files to folder task. Could you take a look please?